docs/team/venuslimm.md

```diff
@@ -10,35 +10,57 @@ It provides a centralised platform for them to manage their work, contacts, and
 
 Given below are my contributions to the project.
 
-* **New Feature**: to be added soon
-  * What it does: to be added soon
-  * Justification: to be added soon
-  * Highlights: to be added soon
-  * Credits: to be added soon
-
-* **New Feature**: to be added soon
-  * What it does: to be added soon
-  * Justification: to be added soon
-  * Highlights: to be added soon
-  * Credits: to be added soon
+* **New Feature**: Added a `addevent` command that adds an event. ([#33](https://github.com/AY2223S2-CS2103-W16-3/tp/pull/33))
+  * What it does: 
+    * Allows user to add an event with the given name, start date time, and end date time.
+  * Justification: 
+    * With this feature, the user is able to manage their events on this application by simply adding the events they are involved in or the events they wish to keep track.
+
+* **New Feature**: Added a `sortevent` command that sorts the current list of events displayed. ([#69](https://github.com/AY2223S2-CS2103-W16-3/tp/pull/69))
+  * What it does:
+    * Allows user to sort the current list of events displayed on the application.
+  * Justification: 
+    * With this feature, the user is able to handle their list of events efficiently in a logical, neater, and consistent order, by sorting the events by either their names or date times.
+  * Highlights: 
+    * Users are given the choice to sort the events based on the events' names (in ascending/descending order), or events' start date times (in ascending order), or events' end date times (in ascending order).
 
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s2.github.io/tp-dashboard/?search=venuslimm&breakdown=true)
 
 * **Enhancements implemented**:
-  * to be added soon
+  * Updated the application's user interface.
```

Comment:
Perhaps consider elaborating this to be a bit more specific here. Can also consider linking your pr next to this too!